### ISSUE TITLE
Guidelines on Reference Management Best Practices

### ISSUE THREAD

**Issue by odontaster** [ices-eg/DIG#278]
The Reference Management Group would like to have some Best Practices reference document in order to better handle some referencing issues. We ask DIG for guidance on this best practices document, which should also include a workflow with competencies and roles of the different involved stakeholders.

**Comment by sjurl** [comment]
If any participant has examples of best practices that can be shared it would be nice if they were made available to the group.

@odontaster set up doodle for a first meeting.

**Comment by HenrikK-N** [comment]
**Conclusion on meaningful versus non-meaningful codes**

@sjurl Regarding your comment on 24 Feb: "If any participant has examples of best practices that can be shared it would be nice if they were made available to the group."

I have written the two documents ‘2022 02 01 Using meaningful codes for information.docx’ and ‘2022 05 09 Meaningful versus Non-meaningful codes.docx’ with arguments for using meaningful codes. See the sharepoint DIG 2022 under Working documents 'Subgroup_Ref...'.
One other document the IOC ‘Ocean Data Standards’ ([IOC/2019/MG/54 Vol.4](https://www.iode.org/index.php?option=com_oe&task=viewDocumentRecord&docID=25099)) was emailed around to the group from one in the group (I just uploaded it to the DIG 2022 sharepoint). This document also support using meaningful codes (in the IOC document called label or abbreviation - both meaningful).

As written in the minutes from the 11 May 2022 (see DIG 2022 Sharepoint site):
"Apart from that, the subgroup agreed that regarding the discussion on meaningful versus non meaningful codes, a form of **compact and meaningful code would be preferred**, however there will be need for further development of these specific issues in the guidelines."

**Comment by HjalteParner** [comment]
@HenrikK-N I think you have misread the document on the NERC vocabularies. The label (whether being the preferred label or the alternative label) isn't what we call 'code' in our vocabulary. Our 'code' is the NERC concept 'key' which indeed is more or less meaningless. And the concept in the document and the NERC vocab is exactly that the key is machine readable and not human readable. The NERC prefLabel is our 'Description' and NERC Definition could be our 'long description' (which we selfdom use to my knowledge).

As such I think you draw the wrong conclusion and in fact the quite the opposite conclusion given in the ocean data standard document on how vocabularies should be build and used.

That being said I don't think there's any harm in using what you call 'meaningful' codes, if it's possible and makes sense for the user group. However the code should as a principle be considered what you call non-meaningless and as such machine readable as a main.

Just to put it into context, you can compare your https://vocab.ices.dk/?CodeID=130498 with the NERC http://vocab.nerc.ac.uk/collection/L05/current/62/ where the code/key respectively is TBB/62 for a beam trawl

**Comment by mehdiabbasi** [comment]
Please read this article that I think will help us to make a clear guideline for ICES vocab [https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009041](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009041) and also this article [https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001414](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001414) about the identifier specifically this section **Lesson 4. Avoid embedding meaning or relying on it for uniqueness** . As @HjalteParner 
 also mentioned, Code in ICES vocab is identifier [https://vocab.ices.dk/services/rdf/collection/IC_GearType/TBB](https://vocab.ices.dk/services/rdf/collection/IC_GearType/TBB)

**Comment by sjurl** [comment]
@odontaster Update for the September 2022 meeting

**Comment by sjurl** [comment]
* Updated draft of guidelines planned for february DIG meeting
   *  Meeting with subgroup before february meeting

**Comment by odontaster** [comment]
Nothing has happenned since the last DIG. I will put this in the RMG agenda to make sure it moves forward.

**Comment by sjurl** [comment]
Decision tree as well as formal way to generate codes. something along the lines of [https://ices-library.figshare.com/articles/report/ICES_CRR_and_TIMES_Author_Guidelines/21608469](https://ices-library.figshare.com/articles/report/ICES_CRR_and_TIMES_Author_Guidelines/21608469 )

**Comment by odontaster** [comment]
The document is currently being reviewed by RMG and when it is internally accepted we will circulate to the broader group for comments.
It will be done by correspondence if possible.

**Comment by sjurl** [comment]
Draft ready by friday 8th december to be circulated to full group for feedback. 
Aim to have a first version ready for all of DIG by february meeting

**Comment by odontaster** [comment]
Draft agreed by ICES secretariat members of the subgroup is ready.
We ask the rest of the subgroup members to review and comment before DIG meeting in May. Adriana will send email now and some reminder half way.
Depending on the magnitude of changes or comments we might ask for a small session in DIG to work on the document.
If review and comments are quick, we might ask for approval in May.
Final approval should not be later than autumn DIG meeting.

**Comment by sjurl** [comment]
Set date for review finalization, short meeting early May with aim to finalize document for DIG May meeting

**Comment by davidcurrie2001** [comment]
@odontaster has there been any further progress on this?  Do you think you will need some time during the DIG meeting?

**Comment by odontaster** [comment]
Hi @davidcurrie2001 
I just sent a reminder for comments. I hope we can get a bit further before DIG, but maybe would be good to set up some short session, maybe 1 hour or so, to discuss and advance.
In my timeline this document should be aproved the latest by DIG Q4. 
I can only make it the first day of DIG, so I would ask you to put it in on Tuesday agenda.
Thanks!

**Comment by davidcurrie2001** [comment]
@odontaster In May we said "_The subgroup will present the final version to DIG for adoption during its September 2024 online meeting_".  Do you have a final version we can circulate for review before our online meeting next week?

**Comment by odontaster** [comment]
@davidcurrie2001 yes I do, working on the last comments these days, will send the link by thursday, hope that is ok!
Thanks!

**Comment by odontaster** [comment]
https://community.ices.dk/Committees/DIG/DIG%202024/02.%20Background%20documents/ICES%20guidelines%20for%20reference%20requests_Sept2024.docx

Will prepare a small presentation/resume of the document. 5 minutes tops for the newcomers, is that ok @davidcurrie2001 ?

**Comment by davidcurrie2001** [comment]
@odontaster -that's great - thanks.

**Comment by davidcurrie2001** [comment]
@odontaster I've moved the document to https://community.ices.dk/Committees/DIG/DIG%202024/September%202024%20Online%20Meeting/ICES%20guidelines%20for%20reference%20requests_Sept2024.docx so that it is kept with with other documents for the September meeting.

**Comment by davidcurrie2001** [comment]
@odontaster presented the document at the September DIG online meeting.  We will leave it open for any final comments until 20/9/24.

**Comment by davidcurrie2001** [comment]
@odontaster - what's the latest status of these guidelines?

**Comment by odontaster** [comment]
@davidcurrie2001 sorry for my late reply. It is in the publication process, will post the DOI here as soon as it is online.

**Comment by odontaster** [comment]
https://doi.org/10.17895/ices.pub.28724003

**Comment by davidcurrie2001** [comment]
Related to #524, #512.
